src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -27,10 +27,10 @@ void AudioMixer::Init(const char* device)
     Close();
 
     SDL_AudioSpec want = {};
-    want.freq = 22050;
+    want.freq = 48000;
```

Comment:
Shouldn't we interrogate the device for the wanted frequency? I think hardcoding it either way will result in other players starting to report audio crackling now.

Reply:
It works the way you describe. `want` is what we requested, but SDL will return what’s actually possible in the `have` variable, which is what we save and work with.